src/main/java/org/sonar/plugins/jacoco/JacocoSensor.java

```diff
@@ -51,20 +52,23 @@ public void execute(SensorContext context) {
   void importReports(ReportPathsProvider reportPathsProvider, FileLocator locator, ReportImporter importer) {
     Collection<Path> reportPaths = reportPathsProvider.getPaths();
     if (reportPaths.isEmpty()) {
-      LOG.debug("No reports found");
+      LOG.info("No report imported, no coverage information will be imported by JaCoCo XML Report Importer");
       return;
     }
 
+    String fileList = reportPaths.stream().sorted().limit(MAX_LOGGED_FILE_NAMES).map(Path::toString).collect(Collectors.joining(";"));
+    if (reportPaths.size() > MAX_LOGGED_FILE_NAMES) {
+      fileList += ";... ";
+    }
+    
+    LOG.info("Importing {} report(s): {}", reportPaths.size(), fileList);
```

Comment:
I wanted to list only the correctly imported reports, but it's not trivial, problems can come from multiple locations, we can not know in this method if it was the case without refactoring many places.

I chose an alternative behavior: list existing paths that we will be imported here.
If a report can not be imported, for any reason, a "warn" log will be displayed.
Otherwise, nothing more will be logged.